ftw/simplelayout/browser/client/dist/simplelayout.js

```diff
@@ -635,6 +635,13 @@ define('app/simplelayout/Block',[
       return this;
     };
 
+    this.markLinks = function() {
```

Comment:
As @maethu asked: is `simplelayout.js` the right place for this function?
It seems to me that `main.js` would be more apropriate.
@shylux has this impact on the test ability? What do you think about that?